Update paid-time-off.md

natalie-hub opened the pull request:
Changed the calendar name to "Time Away" and added a note that this cal can be used for full days away, including time for off-sites, conferences or meet-ups.

<!-- Thank you for contributing a pull request! Here are a few tips to help you:

1. Learn how to update the handbook: https://handbook.mattermost.com/company/how-to-guides-for-staff/how-to-update-handbook
2. If this is your first contribution, please sign the Contributor License Agreement: http://www.mattermost.org/mattermost-contributor-agreement/
   - Once signed, you will be added to the Mattermost Approved Contributor List: https://docs.google.com/spreadsheets/d/1NTCeG-iL_VS9bFqtmHSfwETo5f-8MQ7oMDE5IUYJi_Y/pubhtml?gid=0&single=true
   - If you included your mailing address, you may receive a Limited Edition Mattermost Mug as a thank you gift after your pull request is merged: https://forum.mattermost.org/t/limited-edition-mattermost-mugs/143
-->

#### Summary
<!--
A description of what this pull request does.
-->

jasonblais commented:
Thanks @comharris! If it looks good, you can merge this PR as well 👍

comharris commented on source/leadership/company-v2mom.md:
```diff
@@ -0,0 +1,37 @@
+
+## Values
+
+- Best people for function and culture - High performance, remote first
```
Wondering if this should be `remote-first`? (I know it's verbatim from the doc, but we have `remote-first` elsewhere here)

jasonblais replied:
Good catch, updated

natalie-hub committed:
Update paid-time-off.md

Changed the calendar name to "Time Away" and added a note that this cal can be used for full days away, including time for off-sites, conferences or meet-ups.

jasonblais committed:
Fix typo

jasonblais reviewed (approved):
Thanks!

it33 commented:
Nice change!
